RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -113,6 +107,107 @@ void auto_SITCourse()
 	}
 }
 
+boolean auto_havePayPhone()
+{
+	static item phone = $item[closed-circuit pay phone];
+	return auto_is_valid(phone) && item_amount(phone) > 0;
```

Comment:
```suggestion
	return auto_is_valid($item[closed-circuit pay phone]) && item_amount($item[closed-circuit pay phone]) > 0;
```
why `static`? Nothing is being evaluated here so it gives no benefit.

Reply:
dunno! Copy/pasted it from some other iotm and didn't notice it was static. Agree though, no point here